src/OpenLoco/src/Objects/CurrencyObject.cpp

```diff
@@ -79,7 +79,7 @@ namespace OpenLoco
         _characterWidths[Font::large + 131] = currencyElement->width + 1;
 
         auto& drawingCtx = Gfx::getDrawingEngine().getDrawingContext();
-        drawingCtx.drawStringCentred(rt, x, y - 9, Colour::black, StringIds::object_currency_big_font);
+        drawingCtx.drawStringCentred(rt, Ui::Point(x, y - 9), Colour::black, StringIds::object_currency_big_font);
```

Comment:
after this the next project will be to change the drawPreviewImages to take a point :-P

Reply:
Haha, yes, for sure. Same with drawRect etc.